### DISCUSSION TITLE
Welcome to minimind Discussions!

### DISCUSSION THREAD

**Discussion by jingyaogong** [jingyaogong/minimind#28] [Announcements]
## 👋 欢迎！
  我们使用讨论区作为与社区其他成员交流的地方。我们希望您：
  * 提出您好奇的问题。
  * 分享您的想法。
  * 与其他社区成员互动。
  * 欢迎新成员并保持开放的心态。记住，这是一个我们一起建立的社区 💪。
 
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by WangRongsheng** [comment]
https://github.com/jingyaogong/minimind-v 建议使用Minimind作为LLM，Visual Encoder可以使用其它的

**Comment by Frankyfliu** [comment]
Hi Jingyaogong,

I hope this message finds you well! I’ve sent you a private message via Bilibili regarding your project on GitHub. When you have a moment, please check it out.

Looking forward to hearing from you!

Best regards,
Frank
